src/controllers/user.js

```diff
@@ -82,7 +80,13 @@ export const updateById = async (req, res) => {
         message: 'Cohort could not be found'
       })
     }
-    return sendDataResponse(res, 200, { test: updatedProfile })
+
+    const updatedProfile = await userToUpdate.update()
+    userToUpdate.cohortId = cohortId
```

Comment:
Can you confirm the order of these two lines is accurate / explain how this is working?

I'd normally expect something like this:
```js
const updatedProfile = await userToUpdate.update({
  where: { id: userId },
  data: { cohortId: cohortId }
})
```
Prisma might be able to support the following - but calling update after a change has happened.
```js
userToUpdate.cohortId = cohortId
const updatedProfile = await userToUpdate.update()
```

Reply:
changed the order of the lines to declare the change before updating